tests/unit/lms/services/grouping_test.py

```diff
@@ -29,42 +29,28 @@ def test_upsert_with_parents_inserts(self, svc, db_session):
 
         assert db_session.query(CanvasGroup).one() == test_grouping
 
-    def test_upsert_with_parents_updates(self, svc, db_session):
+    def test_upsert_with_parent_updates_existing_groupings(self, svc, db_session):
```

Comment:
Made the test name a little longer and more explicit, this should be clearer I think (and still fits on one line)